stock_picking_report_product_sticker/models/stock_picking.py

```diff
@@ -0,0 +1,35 @@
+from odoo import api, fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    show_product_stickers = fields.Boolean(
+        help="Show Product Stickers on pickings of this type.",
+        compute="_compute_show_product_stickers",
+        store=True,
+    )
```

Comment:
Set it writable here, so it becomes more obvious that this compute method is replacing an onchange:

```suggestion
    show_product_stickers = fields.Boolean(
        help="Show Product Stickers on pickings of this type.",
        compute="_compute_show_product_stickers",
        store=True,
        readonly=False,
    )
```

Otherwise, it seems like an overkill for a related field. But I see that what you wanted to do is to get a default value from the `stock.picking.type`, but still let users override it at the `stock.picking` level, right?

Reply:
Right

Reply:
OK! then please follow the suggestion and set it as writable at the model level.